frontend/pages/data-card/[dataCardId].tsx

```diff
@@ -52,16 +56,19 @@ export default function DataCard() {
   return (
     <>
       <Title text={dataCard ? dataCard.name : 'Loading...'} />
-      {isDataCardLoading && <Loading />}
-      {dataCard && (
-        <PageWithTabs
-          title={dataCard.name}
-          subheading={`ID: ${dataCard.id}`}
-          tabs={tabs}
-          requiredUrlParams={{ dataCardId: dataCard.id }}
-          titleToCopy={dataCard.name}
-          subheadingToCopy={dataCard.id}
-        />
+      {!dataCard || isDataCardLoading || isCurrentUserLoading || isUiConfigLoading ? (
+        <Loading />
+      ) : (
+        dataCard && (
```

Comment:
Nesting additional conditional logic within a ternary is generally considered something to avoid, simply for the fact that it becomes difficult to read